openlr/openlr_model.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include "geometry/latlon.hpp"
+#include "geometry/point2d.hpp"
+
+#include "std/limits.hpp"
+
+namespace openlr
+{
+enum class FormOfAWay
```

Comment:
This is the name of the type in documentation. I would like to leave it as is if you don't mind.